definitions/npm/apollo-link-http_v1.2.x/flow_v0.28.x-/apollo-link-http_v1.2.x.js

```diff
@@ -0,0 +1,20 @@
+declare module "apollo-link-http" {
+  declare export interface FetchOptions {
+    uri?: string;
+    //todo change to type of Global['fetch']
+    fetch?: any;
```

Comment:
What is `Global['fetch']`? What is the type of `Global['fetch']`? (This is how the value is assigned in the source.)

Is it OK to use `any` here? Or `mixed`?

Reply:
`any` is fine, as it matches the `ts` definition.